src/types/index.ts

```diff
@@ -74,12 +74,11 @@ export type PreviewResponse = {
   rows: ResultRow[];
 };
 
-export type CheckNullAidsResponse = {
-  columns: ResultColumn[];
-  rows: ResultRow[];
+export type HasMissingValuesResponse = {
+  hasMissingValues: boolean;
```

Comment:
This feels a bit redundant. Calling the structure `CheckForMissingValuesResponse` avoids that, but is a bit verbose.

Reply:
alternative is to just return a plain `bool/boolean` (`true/false`). This gives up on the convention to keep all response/request structured. I coded it, it just works with 

```
type HasMissingValuesResponse = bool
```

and similar changes where needed. WDYT?

Reply:
That sounds good too.